### PR TITLE
feat: Add link prop icon position

### LINKED ISSUE #383: New Icon with angle props for Link

### PR DESCRIPTION
**Pull request by gusliedke**
Type: patch

Fixes #383

image

### CODE REVIEW

File: src/components/Atoms/Icons/Arrow.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  transform: ${({ direction }) => `rotate(${angle[direction]})`};
+`;
+
+const Arrow = ({
+  colour, theme, size, direction, ...rest
+}) => (
+  <Icon
+    direction={direction}
+    width={size}
+    heiht={size}
```
Comment (AndyEPhipps) [inline src/components/Atoms/Icons/Arrow.js:22]: Typo

File: src/components/Atoms/Link/Link.js [hunk 2]
```diff
@@ -64,6 +66,8 @@ Link.propTypes = {
   /** Link url */
   href: PropTypes.string.isRequired,
   children: PropTypes.node.isRequired,
+  /** Icon first */
```
Comment (AndyEPhipps) [inline src/components/Atoms/Link/Link.js:69]: lol, probably don't need this comment?

File: src/components/Atoms/Icons/Arrow.js (hunk 1 above)
Comment (gusliedke) [reply src/components/Atoms/Icons/Arrow.js:22]: opps all icons had same typo. thank!

File: src/components/Atoms/Link/Link.js (hunk 2 above)
Comment (gusliedke) [reply src/components/Atoms/Link/Link.js:69]: nah, just as other props had.. this comment will show as description in styleguide frontend .. but field name is explanatory anyway

Comment (AndyEPhipps) [reply src/components/Atoms/Link/Link.js]: Gotcha

**Review by AndyEPhipps — APPROVED** [review]

**Pull request merged**